mopidy_spotify/library.py

```diff
@@ -63,20 +63,13 @@ def _lookup_album(self, sp_link):
     def _lookup_artist(self, sp_link):
         sp_artist = sp_link.as_artist()
         sp_artist_browser = sp_artist.browse(
-            type=spotify.ArtistBrowserType.NO_TRACKS)
+            type=spotify.ArtistBrowserType.FULL)
```

Comment:
If you only do this change do you still get the performance speed-up without sacrificing the loss of fucntionality?